input/drivers_joypad/sdl_joypad.c

```diff
@@ -447,10 +475,20 @@ static void sdl_joypad_poll(void)
       switch (event.type)
       {
          case SDL_JOYDEVICEADDED:
+            RARCH_WARN("[SDL_JOYPAD] hotplug added: %d\n", event.jdevice.which);
             sdl_pad_connect(event.jdevice.which);
             break;
          case SDL_JOYDEVICEREMOVED:
```

Comment:
Should we consider implementing [SDL_ControllerDeviceEvent](https://wiki.libsdl.org/SDL2/SDL_ControllerDeviceEvent)s `SDL_CONTROLLERDEVICEADDED` and `SDL_CONTROLLERDEVICEREMOVED`?